unfoldingWord/translationCore
# Project Import Check Container Fixes

#### This pull request addresses:
This PR improves the workflow and implementation of checking actions.

#### How to test this pull request:
Test this PR by going through the normal workflow of the project import check


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2411)
